praw/objector.py

```diff
@@ -61,6 +61,11 @@ def objectify(self, data):
                 or {'id', 'name', 'permissions'}.issubset(set(data.keys()))):
             parser = self.parsers[self._reddit.config.kinds['redditor']]
             return parser.parse(data, self._reddit)
+        elif isinstance(data, dict) and (
+                {'conversation', 'messages', 'user', 'modActions'}
+                .issubset(set(data.keys()))):
```

Comment:
Yes, if I'm understanding you right. The only keys in the response are `conversation`, `messages`, `user`, and `modActions`.